NOTES.txt

```diff
@@ -0,0 +1,49 @@
+
+robin's github page:
+https://github.com/itsleeds
+
+ITSLeeds is a "team"
+	R4TS is a repo (project) (like a sourceforge project)
+	GitHubNotes is another repo (project) containing these notes
+	each repo can have different owners
+	accounts for individuals in it
+	ITSLeeds/GitHubNotes is owned and must be approved by Robin
+
+GitHub is not free software - it is a propriatory, for-profit web site
```

Comment:
partly I would say + they're definitely good guys, who could be bad with an octocat as a mascot: https://www.flickr.com/photos/cjmartin/15418062046